src/brasil/gov/vlibrasvideo/static/main.js

```diff
@@ -2,22 +2,29 @@ $(function() {
   var VLibrasVideo = (function() {
     function VLibrasVideo() {
       this.$a = $('#viewlet-vlibrasvideo a');
+      this.closeTitle = this.$a.attr('data-close-title');
       this.state = this.$a.attr('class');
-      this.$a.on('click', this, function(e) {
-        var self = e.data;
-        e.preventDefault();
-        if (self.state === 'ready') {
-          self.state = 'playing';
-          var html = "<div id='vlibrasvideo-player'>" +
-            "<iframe width='560' height='315' src='" + (self.$a.attr('href')) + "?autoplay=1' frameborder='0' allowfullscreen></iframe>" +
-          "</div>";
-          $('body').append(html);
-        } else if (self.state === 'playing') {
-          self.state = 'ready';
-          $('#vlibrasvideo-player').remove();
-        }
-      });
+      this.$a.on('click', $.proxy(this.iconClick, this));
+      $('body').on('click', '#vlibrasvideo-player .close', $.proxy(this.closeClick, this));
     }
+    VLibrasVideo.prototype.iconClick = function(e) {
+      e.preventDefault();
+      if (this.state === 'ready') {
+        this.state = 'playing';
+        var html = "<div id='vlibrasvideo-player'>" +
+          "<a href='#' class='close' title='" + this.closeTitle + "'>" + this.closeTitle + "</a>" +
+          "<iframe width='560' height='315' src='" + (this.$a.attr('href')) + "?autoplay=1' frameborder='0' allowfullscreen></iframe>" +
```

Comment:
you are right, this is the [list of valid attributes](https://github.com/simplesconsultoria/sc.embedder/blob/master/src/sc/embedder/config.py#L8-L35), I removed the frameborder into javascript and it worked fine